scripts/announce_failing_screenshots.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env zsh
+
+set -euo pipefail
+
+SCREENSHOT_URL=$1
+
+if curl --output /dev/null --silent --head --fail "$SCREENSHOT_URL"; then
+  echo  "found screenshots at  $SCREENSHOT_URL "
+else
+  curl --output /dev/null --silent $Failed_SCREENSHOTS_TEAMS \
+  -H 'Content-Type: application/json' \
+  --data-binary @- << EOF
+{
+  "@type": "MessageCard",
+  "@context": "http://schema.org/extensions",
+  "themeColor": "0076D7",
+  "summary": "screenshots have failed!",
+  "sections": [{
+      "activityTitle": "screenshots have failed!",
```

Comment:
```suggestion
      "activityTitle": "Screenshots have been failed!",
```

Reply:
I think have failed is correct

Reply:
Hmm I am not an expert with English Grammar but this sounds strange to me. `"activityTitle": "Screenshots have failed!",` would be my idea.

Reply:
Edited my suggestion, totally agree - I realised the mistake on the second read.